### ISSUE TITLE
Cannot find FileSaver.js with in-repo-engine

### ISSUE THREAD

**Issue by GCheung55** [cogniteev/ember-cli-file-saver#3]
I have an in-repo-engine that is trying to load `ember-cli-file-saver` but am seeing:

```
no such file or directory, open '/somerepo/tmp/source_map_concat-input_base_path-yZSiVeUk.tmp/undefined/FileSaver/FileSaver.js'
```

Looking around it seems that switching from loading FileSaver via bower to npm is the right course of action.

**Comment by panthony** [comment]
Should be fixed with `1.0.0` !

**Comment by GCheung55** [comment]
Thank you!
